test/specs/faceting/tests/01.faceting.js

```diff
@@ -1339,7 +1339,33 @@ exports.execute = function (options) {
                     });
                 });
 
-                it ("should return the list of rownames in entity-mode.", function (done) {
+                it ("should return the list of rownames (moustache templates) in entity-mode.", function (done) {
```

Comment:
As we discussed since this is not needed, if it's easy please remove the changes that you made here.